backend/backend-application/src/main/resources/application.yml

```diff
@@ -28,3 +28,11 @@ mongeez:
 ext:
   google-maps:
     api: AIzaSyCluVkO-_MfzcNku1aocMtQp8ua8oUSE0o
+  restpack:
+#    apiToken: O0Bet7ddVPDE5EUMP9s0zmLALUCZ09j5ZUxdEDCURRphtGIv
```

Comment:
Remove

Reply:
But why? It may be used as "backup" token in case first one's limit is met